packages/css-library/tokens/color.json

```diff
@@ -267,6 +267,12 @@
   "vads-color-action-border-base-active-on-dark": {
     "*": { "value": "{uswds-system-color-gray-30}"}
   },
+  "vads-color-feedback-foregound-success-on-dark": {
```

Comment:
I think `foregound` should be `foreground`?

Reply:
Nice catch! Fixed.